### PR TITLE
Feature Support --load-balancer-class in Helm Chart(#3629)

### PR DESCRIPTION
**Pull request by YoonHyunWoo**
### Issue

#3629 

### Description

Enables configuration of the --load-balancer-class flag via .Values.loadBalancerClass in the Helm chart

### Checklist
- [ ] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: helm/aws-load-balancer-controller/templates/deployment.yaml [hunk 1]
```diff
@@ -162,6 +162,9 @@ spec:
         {{- if .Values.certDiscovery.allowedCertificateAuthorityARNs }}
         - --allowed-certificate-authority-arns={{ .Values.certDiscovery.allowedCertificateAuthorityARNs }}
         {{- end }}
+        {{- if .Values.loadBalancerClass }}
+        - --defaultLoadBalancerClass={{ .Values.loadBalancerClass }}
```
Comment (oliviassss) [inline helm/aws-load-balancer-controller/templates/deployment.yaml:166]: we do not have `defaultLoadBalancerClass` as controller flag, do you mean `- --load-balancer-class` here?
see the supported configurations here: https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.7/deploy/configurations/

Comment (YoonHyunWoo) [reply helm/aws-load-balancer-controller/templates/deployment.yaml:166]: sry
--load-balancer class is correct
I did not commit after changing the code locally during the test.

I'll fix it soon

**Pull request merged**